st3/sublime_lib/output_panel.py

```diff
@@ -2,12 +2,23 @@
 
 
 class OutputPanel(ViewStream):
-    def __init__(self, window, name):
+    def __init__(self, window, name, *,
+        settings=None,
```

Comment:
Please use visual indentation when there are arguments on the first line.

Don't we run flake8 on this? It should complain here.

Reply:
I haven't automated it yet on my end.